packages/api/src/sequelize/migrations/2024-04-18_00_update-facility-organization.ts

```diff
@@ -0,0 +1,98 @@
+import { DataTypes } from "sequelize";
+import type { Migration } from "..";
+import { FacilityType } from "../../models/medical/facility";
+import { OrganizationType } from "../../models/medical/organization";
+
+const facilityTableName = "facility";
+const organizationTableName = "organization";
+
+// Use 'Promise.all' when changes are independent of each other
+// Docs: https://sequelize.org/api/v6/class/src/dialects/abstract/query-interface.js~queryinterface
+export const up: Migration = async ({ context: queryInterface }) => {
+  return queryInterface.sequelize.transaction(async transaction => {
+    await queryInterface.addColumn(
+      facilityTableName,
+      "oid",
+      { type: DataTypes.STRING },
+      { transaction }
+    );
+    await queryInterface.addColumn(
+      facilityTableName,
+      "facility_number",
+      { type: DataTypes.NUMBER },
+      { transaction }
+    );
+    await queryInterface.addColumn(
+      facilityTableName,
+      "cq_obo_active",
+      { type: DataTypes.BOOLEAN, defaultValue: false },
+      { transaction }
+    );
+    await queryInterface.addColumn(
+      facilityTableName,
+      "cw_obo_active",
+      { type: DataTypes.BOOLEAN, defaultValue: false },
+      { transaction }
+    );
+    await queryInterface.addColumn(
+      facilityTableName,
+      "cq_obo_oid",
+      { type: DataTypes.STRING, allowNull: true },
+      { transaction }
+    );
+    await queryInterface.addColumn(
+      facilityTableName,
+      "cw_obo_oid",
+      { type: DataTypes.STRING, allowNull: true },
+      { transaction }
+    );
+    await queryInterface.addColumn(
+      facilityTableName,
+      "type",
+      {
+        type: DataTypes.ENUM(...Object.values(FacilityType)),
+        defaultValue: FacilityType.initiatorAndResponder,
+      },
+      { transaction }
+    );
+    await queryInterface.addColumn(
+      organizationTableName,
+      "type",
+      {
+        type: DataTypes.ENUM(...Object.values(OrganizationType)),
+        defaultValue: OrganizationType.healthcareProvider,
```

Comment:
let's use a literal here and instead of `FacilityType.initiatorAndResponder` as well, so its a snapshot